readme.md

```diff
@@ -127,6 +128,28 @@ Directories are recursive by default. Files in directories named `fixtures` and
 
 *WARNING: NON-STANDARD BEHAVIOR:* The AVA CLI will always try to find and use your projects local install of AVA. This is true even when you run the global `ava` command. This non-standard behavior solves an important [issue](https://github.com/sindresorhus/ava/issues/157), and should have no impact on everyday use.
 
+## Configuration
+
+All of the CLI options can be configured in the `ava` section of your `package.json`:
+
+```json
+{
+  "ava": {
+    "files": [
+      "my-test-folder/*.js",
+      "!**/not-this-file.js"
+    ],
+    "fail-fast": true,
```

Comment:
Or you could just use objectAssign on cli.flags instead of using the minimist default thing.